code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -228,13 +228,14 @@
 		if(istype(the_reagent, /datum/reagent/medicine))
 			medibonus += 1
 	M.adjustToxLoss(-0.2 * medibonus)
-	M.adjustOrganLoss(ORGAN_SLOT_LUNGS, medibonus ? 2.5/medibonus : 1)
-	for(var/datum/reagent/the_reagent2 in M.reagents.reagent_list)
+	M.adjustOrganLoss(ORGAN_SLOT_LUNGS, medibonus ? 1.5/medibonus : 1)
```

Comment:
if medibonus is always minimum 1 because it includes itself, what's the point of this ternary?

Reply:
if it somehow deletes before the loop is called